rhessys/cycle/surface_daily_F.c

```diff
@@ -189,6 +207,21 @@ void		surface_daily_F(
 	hv = (2.5023e6 - 2430.54 * zone[0].metv.tday) / 1000; /* changed to match hv used in penman monteith */
 	water_density = 1000;	/* density of water in kg/m^3 */
 	
+	double daylength = zone->metv.dayl;
+	double nightlength = SECONDS_PER_DAY - daylength;
+	double day_proportion = daylength / SECONDS_PER_DAY;
+
+	double surface_heat_flux_day = day_proportion * patch->surface_heat_flux;
+	double surface_heat_flux_night = patch->surface_heat_flux - surface_heat_flux_day;
+
+	double rain_duration_day = zone->daytime_rain_duration * day_proportion;
```

Comment:
@naomitague We don't currently track the time of day when sunrise and sunset occur.  Without this, it is not possible to apportion hourly rainfall to either daytime or nighttime.  I think it is out of the scope for this pull request to add this ability.  I have created #81 to capture this need.